sql/index.go

```diff
@@ -71,7 +71,7 @@ type Index interface {
 	IsFullText() bool
 	// Comment returns the comment for this index
 	Comment() string
-	// IndexType returns the type of this index, e.g. BTREE
+	// IndexClass returns the type of this index, e.g. BTREE
```

Comment:
Did you mean to change this doc string? Looks like the function is still named `IndexType`.